docker/env/version

```diff
@@ -1 +1 @@
-0.78
+0.80
```

Comment:
You broke Hydra version 0.80, this is not the way we update the Hydra Docker image. Did you talk with somebody before doing this?

Reply:
also version `0.79` was burnt...
anyhow next time I think it would be clear, when building a version docker for testing a PR,
it should be `0.80_[some suffix to describe it]`, it should be rebuild into a formal version only once the PR is approved.
(which isn't the case yet)

Reply:
> You broke Hydra version 0.80, this is not the way we update the Hydra Docker image. Did you talk with somebody before doing this?

what was broken with v0.80?

yes, I spoke the guys, and as @fruch  mentioned in his comment [here](https://github.com/scylladb/scylla-cluster-tests/pull/2761#issuecomment-704538833), next time, i will create it once getting close to approval (as I expected it to be) or once approved (and then change the version file + build and push the image)...